00-programming-fundamentals/exercises/merge-conflict-remote.md

```diff
@@ -0,0 +1,102 @@
+
+### Building the recipe
+
+Once everyone in your group has completed the baseline setup instructions above you're ready to start building the recipe together.
+
+#### General advice
+
+There are a few things to look out for when working through this activity:
+
+- If you're still unsure about what a merge conflict is, or how to resolve one please review the [Merge Conflicts curriculum](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/00-programming-fundamentals/git-merge-conflicts.md).
+- You can review the [slides](https://docs.google.com/presentation/d/15OFyyoNJevzk7IpdZCP6VwMLju0MVS2pv0qeETLOa6Q/edit?usp=sharing) and [example walkthrough video](https://adaacademy.hosted.panopto.com/Panopto/Pages/Viewer.aspx?id=ec7dbf60-e59b-4317-a4a5-abbe002eb7ea) if you are unsure about how to complete the activity.
+- The end result should be a real, usable recipe. The original recipe files were all complete works from various recipe websites online.
+- If you find that the result of one of your merges is missing critical ingredients or any numbered steps, you should go back and double check that you didn't lose any lines while resolving a merge conflict.
+- Your recipe scraps may have duplicate lines. We've added these in to more accurately portray the full range of potential merge conflicts. **Important**: you should get rid of any duplicate lines while merging.
+- There's no "right" way to do this activity, all of the strategies listed below are valid ways to build the recipe file.
+- Have fun!
+
+#### Recipe building process
+
+Because each member of your group now has a different version of the recipe file in their local repository, building the complete recipe by merging those versions together will result in merge conflicts. To handle those merge conflicts sensibly your group should use the following process to construct the final recipe file:
+
+1. Split your group into two teams of three. (There may be a group of two if your group only has 5 people.)
+2. Each person on a team will create a branch with `git checkout -b BRANCH-NAME`, selecting their own branch name based off the recipe scrap they have, and commit the changes they have made to the recipe.
+3. Each team member will push their changes up to github with `git push origin BRANCH-NAME`.
+4. Both teams then will use the _round robin_ strategy separately.
+    - Each team member will open a **pull request** trying to merge their branch on github into `master`.  **Be very careful to make the PR against your forked repository and NOT AdaGold.**
+    - If there is a report of a merge conflict they will need to:
+     - Pull the current state of master into their branch with `git pull origin master`
+     - Resolve the merge conflicts and commit the result
+     - Push the result up to git hub with `git push origin BRANCH-NAME`
+     - Then attempt to merge their pull request.  If new changes have happened on master they may have to repeat step 1 above. 
```

Comment:
It reads like these should be nested within the above bullet. I'm not sure if adding 2 spaces will fix it or not.
```suggestion
       - Pull the current state of master into their branch with `git pull origin master`
       - Resolve the merge conflicts and commit the result
       - Push the result up to git hub with `git push origin BRANCH-NAME`
       - Then attempt to merge their pull request.  If new changes have happened on master they may have to repeat step 1 above. 
```